src/axom/sidre/core/Group.hpp

```diff
@@ -293,35 +295,38 @@ class Group
    */
   Group* setDefaultAllocator(umpire::Allocator alloc)
   {
-    m_default_allocator_id = alloc.getId();
+    setDefaultAllocator(alloc.getId());
     return this;
   }
+#endif
 
   /*!
    * \brief Set the default umpire::Allocator associated with this Group.
    */
   Group* setDefaultAllocator(int allocId)
   {
+#if !defined(AXOM_USE_UMPIRE)
+    SLIC_ASSERT(allocId == axom::MALLOC_ALLOCATOR_ID);
+#endif
     m_default_allocator_id = allocId;
+    m_default_allocator_id_conduit =
+      axom::ConduitMemory::instanceForAxomId(m_default_allocator_id).conduitId();
     return this;
   }
-#else
-  /*!
-   * \brief Return the ID of the default umpire::Allocator associated with this
-   * Group.
-   */
-  int getDefaultAllocatorID() const { return axom::getDefaultAllocatorID(); }
 
   /*!
-   * \brief Set the default umpire::Allocator associated with this Group.
+   * \brief Reallocate data to View-specific allocators.
+   *
+   * \param [i] viewToAllocatorId A function that returns the allocator
+   *   id to reallocate the View's data to.  Where it returns
+   *   the View's current allocator or axom::INVALID_ALLOCATOR_ID,
+   *   the View is unchanged.
```

Comment:
Please clarify that `axom::INVALID_ALLOCATOR_ID` is used for views that should not be reallocated.

Reply:
As mentioned above, the `reallocateTo` method will be removed altogether.